packages/cardano-services/test/Asset/DbSyncNftMetadataService.test.ts

```diff
@@ -33,12 +33,12 @@ describe('DbSyncNftMetadataService', () => {
     await dbConnection.end();
   });
 
-  it('returns undefined for unexisting asset', async () => {
-    expect(await service.getNftMetadata(unexistingAsset)).toBeUndefined();
+  it('returns null for unexisting asset', async () => {
```

Comment:
```suggestion
  it('returns null for non-existent asset', async () => {
```

Reply:
done in [7b87d1d](https://github.com/input-output-hk/cardano-js-sdk/commit/7b87d1ded0df482d29dd2b581a3baa9035f32d06)